src/platforms/javascript/guides/angular/config.yml

```diff
@@ -0,0 +1,10 @@
+title: Angular
+caseStyle: camelCase
+supportLevel: production
+sdk: "sentry.javascript.browser"
```

Comment:
this defaults to whatevers in javascript I believe so can probably remove it, same for caseStyle/supportLevel/categories

Reply:
Removed caseStyle, supportLevel, and sdk.